R/pkg/R/DataFrame.R

```diff
@@ -883,6 +883,9 @@ setMethod("toJSON",
 #' Save the contents of a SparkDataFrame as a JSON file (\href{https://jsonlines.org/}{
 #' JSON Lines text format or newline-delimited JSON}). Files written out
 #' with this method can be read back in as a SparkDataFrame using read.json().
+#' You can find the JSON-specific options for writing JSON files in
+#' \href{https://spark.apache.org/docs/latest/sql-data-sources-json.html#data-source-option}{
+#' Data Source Option} in the version you use.
```

Comment:
I think this should go to `@param ... additional argument(s) passed to the method.` below